src/docs/deployment/android.md

```diff
@@ -60,29 +62,36 @@ If you have an existing keystore, skip to the next step.
 If not, create one by running the following at the command line:
 
 On Mac/Linux, use the following command:
-```
+
+```terminal
 keytool -genkey -v -keystore ~/key.jks -keyalg RSA -keysize 2048 -validity 10000 -alias key
 ```
+
 On Windows, use the following command:
-```
-keytool -genkey -v -keystore c:/Users/USER_NAME/key.jks -keyalg RSA -keysize 2048 -validity 10000 -alias key
+
+```terminal
+keytool -genkey -v -keystore c:/Users/USER_NAME/key.jks -storetype JKS -keyalg RSA -keysize 2048 -validity 10000 -alias key
 ```
 
-{{site.alert.note}}
+{{site.alert.warning}}
   Keep the `keystore` file private;
   do not check it into public source control.
 {{site.alert.end}}
 
 {{site.alert.note}}
-  The `keytool` command might not be in your path&mdash;it's
-  part of the Java JDK, which is installed as part of Android Studio.
-  For the concrete path,
-  run `flutter doctor -v` and locate the path printed after
-  'Java binary at:'. Then use that fully qualified path
-  replacing `java` (at the end) with `keytool`.
-  If your path includes space-separated names,
-  such as `Program Files`, place quotes around the space-separated
-  names. For example: `/"Program Files"/`
+  * The `keytool` command might not be in your path&mdash;it's
+    part of the Java JDK, which is installed as part of
+    Android Studio.  For the concrete path,
+    run `flutter doctor -v` and locate the path printed after
+    'Java binary at:'. Then use that fully qualified path
+    replacing `java` (at the end) with `keytool`.
+    If your path includes space-separated names,
+    such as `Program Files`, place quotes around the space-separated
+    names. For example: `/"Program Files"/`
+
+  * The `-storetype JKS` tag is only required for Java 9
+    or newer. As of this release, the keystore type defaults
```

Comment:
What does "this release" refer to? Java 9? Or a Flutter release? (I don't understand the tech; you might want to get someone who does understand to check your words.)